catalog/registry/catalog-registry-federationadminservice/src/main/java/ddf/catalog/registry/federationadmin/service/FederationAdminService.java

```diff
@@ -14,12 +14,10 @@
 package ddf.catalog.registry.federationadmin.service;
 
 import java.util.List;
+import java.util.Set;
 
 import ddf.catalog.data.Metacard;
-import ddf.catalog.federation.FederationException;
-import ddf.catalog.source.IngestException;
-import ddf.catalog.source.SourceUnavailableException;
-import ddf.catalog.source.UnsupportedQueryException;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryPackageType;
 
 /**
  * Service that provides the interface to get and add registry metacards
```

Comment:
Add the experimental disclaimer to the class javadoc. `<b> This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library. </b>`  I got the same comment from scott so passing it along.